cmake/modules/SwiftUtils.cmake

```diff
@@ -0,0 +1,6 @@
+
+function(precondition var)
+  if (NOT ${var})
+    message(FATAL_ERROR "Error! Variable ${var} is not set!")
```

Comment:
It can actually be set to `false`, `0` or any other string that CMake treats as false.